src/main/java/com/teentech/hotels/controller/PublicMenuController.java

```diff
@@ -13,7 +13,7 @@
 
 @RestController
 @CrossOrigin
-@RequestMapping("/public/menu")
```

Comment:
Sigur aici s-a stricat ceva, nu cred ca trebuia modificat. Asa cu public/menu era pus pe lista permisiunilor in WebSecurityConfig